HW10/HW10_Interfaces_Abstract_classes/src/Main.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Objects;
+import java.util.HashMap;
+public class Main {
+    public static void main(String[] args) {
+
+       // Створіть кілька користувачів.
+        // Додайте контакти для кожного користувача.
+        // Відправте повідомлення між користувачами.
+        // Виведіть список повідомлень кожного користувача із зазначенням статусу.
+        // Викличте метод для "прочитування повідомлень" від імені користувачів,
+        // які отримали повідомлення та перевірте статус цих повідомлень після прочитання.
+
+        User tania = new User("Tania");
+        User artem = new User("Artem");
+        User mum = new User("Mum");
+
+        tania.addContact("0999999999", artem);
+        tania.addContact("099994444", mum); // Tania - 2 contacts
+        artem.addContact("099999554", tania);
+
+
+        Message firstMessage = new Message("Hello, it's me");
+        Message secondMessage = new Message("Bye Bye");
+        Message thirdMessage = new Message("Spam this is spam");
+        Message fourthMessage = new Message("enemy");
+        Message fifthMessage = new Message("test");
+        Message six = new Message("new text");
+
+        tania.sendMessage(firstMessage, artem);
```

Comment:
What if `.sendMessage(Message message, User recipient)` can be modified to retrieve not an actual message but only a content of it? (the creating of a new Message object can happen inside that method) - this will hide a bit of internal logic so we won't mess with handling messages separately anymore as user is responsible for creating messages right?

Or we can have even different overloads for `.sendMessage()` method with different kinds of input params so you can choose which one you want to choose